scripts/pi-hole/js/auditlog.js

```diff
@@ -134,12 +134,13 @@ function auditUrl(url) {
   add(url, "audit");
 }
 
-$(document).ready(function () {
+$(function () {
   // Pull in data via AJAX
   updateTopLists();
 
   $("#domain-frequency tbody").on("click", "button", function () {
     var url = $(this).parents("tr")[0].textContent.split(" ")[0];
+    // TODO
     if ($(this).context.textContent === " Blacklist") {
```

Comment:
This is deprecated, not sure what the original intention was?

Reply:
Probably something that no longer matters, just remove it.

Reply:
I'd like to wait for @DL6ER since it's probably his code. :)

Reply:
It seems the whole click event is moot currently?

Reply:
Actually, it is needed for the buttons to work. We could just change the target selectors, I think.

If that's the right approach, I can make another patch later.

Reply:
@DL6ER friendly ping how to proceed with this. After I address this, only testing is left.